charts/kubearchive/templates/observability/otel-collector.yaml

```
@@ -0,0 +1,97 @@
+{{- if .Values.integrations.observability.enabled }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: otel-collector-conf
+  labels:
+    app: otel-collector
+data:
+  otel-collector-config: |
+    receivers:
+      otlp:
+        protocols:
+          http:
+            endpoint: 0.0.0.0:4318
+      prometheus:
+        config:
+          scrape_configs:
+            - job_name: 'kubearchive-otel-collector'
+              static_configs:
+                - targets: ['localhost:8888']
+
+    processors:
+      batch:
+
+    exporters:
+      debug:
+      otlphttp:
+        endpoint: "http://grafana-lgtm.{{ .Release.Namespace }}.svc.cluster.local:4318"
+        tls:
+          insecure: true
+
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp]
+          processors: [batch]
+          exporters: [debug, otlphttp]
+        metrics:
+          receivers: [otlp, prometheus]
+          processors: [batch]
+          exporters: [debug, otlphttp]
+        logs:
+          receivers: [otlp]
+          processors: [batch]
+          exporters: [debug]
```

Following the previous suggestion, I would also extract this to `charts/kubearchive/conf/otel-otelcol-config.yaml` and do a change like this:

```suggestion
  otel-collector-config: |-
    {{- tpl ($.Files.Get "conf/otel-otelcol-config.yaml") . | nindent 4 }}
```

---

Done